src/Microsoft.CSharp/tests/Microsoft.CSharp.Tests.csproj

```diff
@@ -2,6 +2,7 @@
   <PropertyGroup>
     <ProjectGuid>{82B54697-0251-47A1-8546-FC507D0F3B08}</ProjectGuid>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <EnableXUnitAnalyzer>true</EnableXUnitAnalyzer>
```

Comment:
Even with my top-level comments, this PR LGTM.  The primary thing I'm concerned about is this one line.  We should be doing this at the repo level at a minimum, and since you say that would cause a significant amount of work to achieve due to lots of warnings across the repo, scoping it to just the critical set of warnings about tests that should be running but aren't; there should be relatively few of those, and we should be able to enable them in one fell swoop.  If it turns out there are way more than "relatively few", that then just highlights we have a bigger issue than we thought and it's worth addressing even more.

Reply:
I get your point. I'll do the necessary to enable these rules and will provide an extensibility point so that assemblies can go the extra mile and enable more rules.

Reply:
Thank you! If you'd prefer I do it, just let me know.

Reply:
Everything's good. I'll do it :)

Reply:
Since this is on at the repo level now, should this line be deleted?